contrib.json

```
@@ -1488,7 +1488,11 @@
             "labels": ["linting", "SublimeLinter", "syntax", "test"],
             "releases": [
                 {
-                    "sublime_text": ">=3092",
+                    "sublime_text": "3092 - 4180",
+                    "tags": "st3-"
+                },
+                {
+                    "sublime_text": ">=4181",
                     "tags": true
```

```suggestion
                    "sublime_text": "3092 - 4180",
                    "tags": "3092-"
                },
                {
                    "sublime_text": ">=4181",
                    "tags": "4181-"
```
I'll likely stick to this strategy that I adoped from deathaxe that more clearly communicates which build a version starts being used for, which is becoming more and more likely to occur since ST moved away from selling major version updates only and especially with regards to syntax packages depending on built-in syntaxes (not relevant here).

---

Make your life easier with `true` for the current/recent cycle, and only mark/freeze a specific version.  That way the tags read natural and look semantic.  🤷‍♀️   It is still possible that such frozen tags are more and more likely in the future, but also that a future PC will allow us to declare `tags: "=1.2.0"`.   🤷‍♂️  But just tell me when you're done.

---

I have created the tags in the format proposed here. Please apply the change as suggested and merge.